src/resources/Restoration/BC.ts

```diff
@@ -0,0 +1,52 @@
+import { RestorationResourceIF } from '@/interfaces'
+import { CorpTypeCd, FilingCodes, RuleIds } from '@/enums'
+import { RestorationSteps } from './steps'
+import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+export const RestorationResourceBc: RestorationResourceIF = {
+  entityType: CorpTypeCd.BC_COMPANY,
+  displayName: GetCorpFullDescription(CorpTypeCd.BC_COMPANY),
+  steps: RestorationSteps,
+  filingData: [{
+    entityType: CorpTypeCd.BC_COMPANY,
+    filingTypeCode: FilingCodes.RESTORATION_FULL
```

Comment:
We'll have to figure something out to handle the filing codes for the other restoration types (limited, limited to full, limited extension).